tests/dragonfly/replication_test.py

```diff
@@ -1263,13 +1263,16 @@ async def delayed_takeover():
 
 @pytest.mark.exclude_epoll
 @pytest.mark.parametrize("master_threads, replica_threads", take_over_cases)
-async def test_take_over_seeder(
-    request, df_factory, df_seeder_factory, master_threads, replica_threads
-):
+async def test_take_over_seeder(df_factory, df_seeder_factory, master_threads, replica_threads):
     master = df_factory.create(
-        proactor_threads=master_threads, dbfilename=f"dump_{tmp_file_name()}", admin_port=ADMIN_PORT
+        proactor_threads=master_threads,
+        dbfilename=f"dump_{tmp_file_name()}",
+        admin_port=ADMIN_PORT,
+        vmodule="rdb_save=1,dflycmd=1,snapshot=1,streamer=3",
```

Comment:
I believe you can remove the vmodule change from this test.
Keeping the vlog changes introduced in this PR for future investigations if needed